docs/reference/tab-widgets/troubleshooting/data/diagnose-unassigned-shards.asciidoc

```diff
@@ -106,6 +106,7 @@ The response will look like this:
       "node_id" : "8qt2rY-pT6KNZB3-hGfLnw",
       "node_name" : "node-0",
       "transport_address" : "127.0.0.1:9401",
+      "roles": [data, data_cold, data_content, data_frozen, data_hot, data_warm, ingest, master, ml, remote_cluster_client, transform]
```

Comment:
nit: (not picked up by the test framework because this example is skipped)

```suggestion
      "roles": [data, data_cold, data_content, data_frozen, data_hot, data_warm, ingest, master, ml, remote_cluster_client, transform],
```

You could also reasonably list fewer roles here, maybe just `data_content, data_hot`, I don't think it helps the docs to have them all.

Reply:
Done